homeassistant/components/energy/validate.py

```diff
@@ -39,12 +39,16 @@
 )
 GAS_USAGE_UNITS = {
     sensor.SensorDeviceClass.ENERGY: (
-        UnitOfEnergy.WATT_HOUR,
+        UnitOfEnergy.GIGA_JOULE,
         UnitOfEnergy.KILO_WATT_HOUR,
         UnitOfEnergy.MEGA_WATT_HOUR,
-        UnitOfEnergy.GIGA_JOULE,
+        UnitOfEnergy.WATT_HOUR,
+    ),
+    sensor.SensorDeviceClass.GAS: (
+        UnitOfVolume.CENTUM_CUBIC_FEET,
```

Comment:
To be clear, this is the 1st out of 2 functionality changes in the PR; the rest are just sorting?

Reply:
Yes indeed - the rest is just sorting